### ISSUE TITLE
Focus-triggering hidden element violates accessibility standards: aria-hidden conflicts with tabindex

### ISSUE THREAD

**Issue by Green-Ysh** [vueComponent/ant-design-vue#7861]
- [x] I have searched the [issues](https://github.com/vueComponent/ant-design-vue/issues)         of this repository and believe that this is not a duplicate.
        
        
### Version
4.2.5

### Environment
macOS 15 、Chrome 128.0.6613.139 、 Vue3.4.37 、Antdv 4.2.5

### Reproduction link
[https://next.antdv.com/components/modal-cn](https://next.antdv.com/components/modal-cn)

### Steps to reproduce
Click on any example of the Modal dialog component on the official website. The issue consistently appears in the browser console.

### What is expected?
Expected outcome: No related errors should occur anymore.

### What is actually happening?
This error frequently appears in the console.
